index.js

```diff
@@ -4,19 +4,22 @@ const fp = require('fastify-plugin')
 const Ajv = require('ajv')
 
 function fastifyResponseValidation (fastify, opts, next) {
+  const isAjvInstance = opts.ajv && opts.ajv instanceof Ajv
```

Comment:
If the user provides its own ajv instance, I would skip all the code below and we need to set the `ajv` var only (no plugins, nor `new Ajv`)

Reply:
Sounds good. Updated code should reflect this now